sdk/search/Azure.Search.Documents/README.md

```diff
@@ -287,7 +289,26 @@ string key = Environment.GetEnvironmentVariable("SEARCH_API_KEY");
 AzureKeyCredential credential = new AzureKeyCredential(key);
 SearchIndexClient client = new SearchIndexClient(endpoint, credential);
 
-// Create the index
+// Create the index using FieldBuilder.
+SearchIndex index = new SearchIndex("hotels")
+{
+    Fields = FieldBuilder.Build(typeof(Hotel)),
+    Suggesters =
+    {
+        // Suggest query terms from the hotelName field.
+        new SearchSuggester("sg", "hotelName")
+    }
+};
+
+client.CreateIndex(index);
+```
+
+In scenarios when the model is not known or cannot be modified, you can
+also create fields explicitly using convenient `SimpleField`,
```

Comment:
The flow seems reasonable, but I think that makes the README example too convoluted and complex / long. I'd still probably want to put an explanation between the code. But, personally, I like the comparison better here: you can use FieldBuilder, or you can do it the "hard" way. The result is the same (or should be).

Reply:
Let's leave it as-is for now, but I'm mostly worried about the main README being too long since SimpleField and friends aren't the mainline scenario.  We can iterate on this in the future.

Reply:
I wrestled with this a bit myself and talked if over with Alex. The idea was that FieldBuilder hopefully works for most people, but to show a concise example (I could even make it shorter: the Hotel model here only has hotelId and hotelName) that you can do it manually as well.